src/main/java/com/emc/object/s3/jersey/S3JerseyClient.java

```diff
@@ -199,6 +191,12 @@ public S3JerseyClient(S3Config config, ClientHandler clientHandler) {
             // S.C. - CHUNKED ENCODING (match ECS buffer size)
             smartConfig.setProperty(ClientConfig.PROPERTY_CHUNKED_ENCODING_SIZE, s3Config.getChunkedEncodingSize());
 
+            // S.C. - CONNECT_TIMEOUT
```

Comment:
This logic (setting the Jersey timeout properties) should be moved to ObjectConfig.toSmartConfig()

Reply:
resolved